libkbfs/key_bundle_v2.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"encoding"
+
+	"github.com/keybase/client/go/protocol/keybase1"
+	"github.com/keybase/go-codec/codec"
+)
+
+// TLFWriterKeyBundleV2 is a bundle of writer keys and historic symmetric encryption
+// keys for a top-level folder.
+type TLFWriterKeyBundleV2 struct {
```

Comment:
It was a while before I realized that WKBV2 corresponded to BRMDV3. That can get confusing fast...can you sync up the version numbers?